sirepo/package_data/static/js/srw.js

```diff
@@ -1462,10 +1468,6 @@ SIREPO.viewLogic('tabulatedUndulatorView', function(appState, panelState, reques
     $scope.whenSelected = updateUndulator;
     $scope.watchFields = [
         ['tabulatedUndulator.undulatorType'], updateUndulator,
-        [
```

Comment:
This seems ok. I just don't know enough about the change.

Reply:
We don't need that watchfield anymore since line 823 -
```javascript
 $scope.$on('modelChanged', function(e, name) {
        if (name == 'undulator' || name == 'tabulatedUndulator') {
```
listens and triggers self.computeBeamParameters